### DISCUSSION TITLE
Add option to delete games using DEL key (without confirmation window)

### DISCUSSION THREAD

**Discussion by mariano-london** [prosthetichead/GarlicPress#50] [Ideas]
I found incredible useful after downloading a romset to go and delete duplicated roms, however is become a bit annoying to reach the mouse to the delete option and then the box

Maybe add this option on setting so people can activate delete without confirmation, so is only used if you are aware of this?

Also a multiple selection would be ace, also after deleting a file cursor goes to the top of the file, if would stay in the previous rom would speed up the process a lot

**Comment by prosthetichead** [comment]
Thanks for the feedback!
I can have a look at the delete idea for sure especially the delete key press. I will also look at the returning the selection back to the previous position instead of the top!

You can already multi select though from the list. Hold down shift to select in a block or ctrl to select individual clicked items. you can also multi select by dragging the mouse with the left button down.

**Comment by mariano-london** [comment]
thanks so much for the quick answer, regarding the multi select I saw the [CTRL] key but i was not very clear I was meaning to add a column where you can tick many roms and then press delete etc 

At the moment Im using the keys to navigate so pressing B I start from there but looking at to comeback to the previous one would be really cool

thanks so much for making such an amazing tool

**Reply by prosthetichead** [reply]
Ahhhh ok that makes sense.  👍

**Reply by prosthetichead** [reply]
I was able to get this working it 

you can now delete using the delete key
turn off the delete prompts in settings
and the next item is selected so you can work your way though the list using the arrow and delete key!

**Reply by mariano-london** [reply]
fantastic that is great thank so much for this amazing work

**Reply by mariano-london** [reply]
I just had to test and is really great the changes

can I bee a bit nitpicking? 

(closing the Update Art window you get back to the start of the list)

would be possible to have the same functionality of "next item select" after updating art?

**Reply by prosthetichead** [reply]
no worries, yeah I just noticed the same thing and it annoyed me also lol

**Reply by mariano-london** [reply]
hahaha that's the thing once you experienced the power of "next item select" you can't settle for less.

fantastic tool you made thanks
